yoti_python_sdk/protobuf/protobuf.py

```diff
@@ -40,13 +43,20 @@ def anchor(data):
         return anchor
 
     def value_based_on_content_type(self, value, content_type=None):
-        if content_type == self.CT_UNDEFINED:
-            raise TypeError('Wrong content type')
-        elif content_type == self.CT_STRING:
+        if content_type == self.CT_STRING:
             return value.decode('utf-8')
         elif content_type == self.CT_DATE:
             return value.decode('utf-8')
-        return value
+        elif content_type == self.CT_JPEG \
+                or content_type == self.CT_PNG:
```

Comment:
Could this be `elif content_type in [self.CT_JPEG, self.CT_PNG]`?

Reply:
I'm easy to be honest, either way is fine I think

Reply:
I think it's fine as is